Lusas_Adapter/Convert/ToBHoM/Loads/ToLoadCombination.cs

```diff
@@ -55,14 +55,15 @@ public static partial class Convert
                 factoredLoadcases.Add(factoredLoadcase);
             }
 
+            long adapterNameId = lusasLoadCombination.getID();
+
             LoadCombination loadCombination = new LoadCombination
             {
                 Name = GetName(lusasLoadCombination),
-                Number = lusasLoadCombination.getID(),
+                Number = (int)adapterNameId,
```

Comment:
As a way to future proof this consider using:

Number = Convert.ToInt32(Math.Max(Math.Min(adapterNameId, int32.MaxValue), int32.MinValue))

This will also stop errors occurring in situations where the long is greater than the (int) can cast to.

Reply:
Added with warning messages.